rts build for windows is broken

Here is build output:

related commit: https://github.com/idris-lang/Idris-dev/commit/6dafd1880aa6a6cf934bc8812dd336be26f8bb66
``` shell
"make"[1]: Entering directory `/c/H/P/Idris-dev/rts'
cc -fPIC -O2 -Wall -DHAS_PTHREAD -DIDRIS_ENABLE_STATS  -I/usr/local/include  -DIDRIS_TARGET_OS="\"windows\"" -DIDRIS_TARGET_TRIPLE="\"mingw32\""   -c
-o idris_rts.o idris_rts.c
idris_rts.c:1:0: warning: -fPIC ignored for target (all code is position independent) [enabled by default]
 #include <stdlib.h>
 ^
In file included from idris_rts.h:13:0,
                 from idris_rts.c:10:
c:\mingw\lib\gcc\mingw32\4.8.1\include\emmintrin.h:31:3: error: #error "SSE2 instruction set not enabled"
 # error "SSE2 instruction set not enabled"
   ^
In file included from idris_rts.c:10:0:
idris_rts.h:77:9: error: unknown type name '__m128i'
         __m128i* bits128p;
         ^
idris_rts.c: In function 'MKB8x16const':
idris_rts.c:326:53: error: '__m128i' undeclared (first use in this function)
     Closure* cl = allocate(sizeof(Closure) + sizeof(__m128i), 1);
                                                     ^
idris_rts.c:326:53: note: each undeclared identifier is reported only once for each function it appears in
idris_rts.c:329:34: error: expected expression before ')' token
     cl->info.bits128p = (__m128i*)ALIGN((uintptr_t)cl + sizeof(Closure), 16);
                                  ^
...
```

--- a/config.mk
+++ b/config.mk
@@ -2,7 +2,7 @@ CC              ?=cc
 CABAL           :=cabal
 # IDRIS_ENABLE_STATS should not be set in final release
 # Any flags defined here which alter the RTS API must also be added to src/IRTS/CodegenC.hs
-CFLAGS          :=-O2 -Wall -DHAS_PTHREAD -DIDRIS_ENABLE_STATS $(CFLAGS)
+CFLAGS          :=-O2 -Wall -DHAS_PTHREAD -DIDRIS_ENABLE_STATS -msse2 $(CFLAGS)
 
 #CABALFLAGS	:=
 ## Disable building of Effects

